scripts/repository.lic

```diff
@@ -1382,10 +1403,46 @@ elsif (cmd[0] =~ /^download$/i)
   if list.empty?
     echo "error: file not found"
   elsif list.length > 1
+    names = list.map { |row| row[0] }
     list.unshift(headers)
-    respond "\nMultiple files match: be more specific\n\n#{format_list.call(list)}\n\n"
+    puts "\nMultiple files match: be more specific\n\n#{format_list.call(list, 'download')}\n\n"
+    if names.length != names.uniq.length
+      respond ""
+      puts Lich::Messaging.msg_format("info", " You can specify a specific author by adding --author=bob to your command or game by adding --game=gs to the command. For example:")
+      puts Lich::Messaging.msg_format("info", "  ;repo download awsomeScript --author=bob")
+      puts Lich::Messaging.msg_format("info", "  ;repo download awsomeScript --game=gs")
```

Comment:
do NOT use `puts` as it has known issues with terminal based FEs. In order to send actual XML, change the `respond` to `_respond`